src/askui/telemetry/utils.py

```diff
@@ -0,0 +1,41 @@
+import uuid
+
+
+def map_guid_to_uuid4(guid: str | bytes) -> str:
```

Comment:
Do we realy need this function? Maybe I have to less knowledge about the specifications. I thought you can pass a GUID directly in the uuid.UUID4 function. 

But maybe I'm wrong.

Reply:
I use it to derive an anonymous user id from the hashed device guid (that is why the input is actually no guid and the name of the function misleading, updated it). I changed the implementation after discussion with @mlikasam-askui to only use this function for deriving the anonymous id but not the device id to keep the implementations across our components simple.